core/templates/dev/head/css/oppia.css

```diff
@@ -352,6 +352,12 @@ p:last-child {
   margin: 0px;
 }
 
+/* This will prevent the enlargment of the text that is nested inside the p
```

Comment:
This is too broad. It affects all `p span` tags, not just RTE ones. In addition using !important typically indicates a problem with the approach. 

Can you use a more specific CSS rule that is only scoped to the RTE viewer component? That may make it possible for you to do this without !important.

Reply:
I think, it would affect a lot of things but it won't break any of them. The nested em size units add up as we go deeper. The `font-size: 1em`, removes this addition and leads to inherit the size from parent itself. I don't think we need a different size in span tag (which is different from that in p tag). Does this sounds good?

There is inline css which is being introduced with the span tag. Can we override that css by making it more specific without using !important? I guess, no. I referred this for the same: https://stackoverflow.com/questions/16813220/how-can-i-override-inline-styles-with-external-css/16813263.

Thanks!

Reply:
Hm, I think this means we really do need to figure out where the inline CSS is coming from. I added a comment on the original issue thread. Thanks for the explanation!